bedrock/mozorg/templates/mozorg/plugincheck.html

```diff
@@ -54,6 +56,8 @@
       {{ download_firefox(small=True) }}
       <h1 class="large">{{_('Check Your Plugins')}}</h1>
       <p class="preamble">{{_('Keeping your plugins up to date helps Firefox run safely and smoothly.')}}</p>
+      {% set share_urls = { 'twitter': 'http://mzl.la/17aqnLZ', 'googleplus': 'http://mzl.la/1FuAhTS', 'facebook': 'http://mzl.la/1z4Noqm' } -%}
+      {{ share_cta(_('Share'), share_urls, _('Plugins don’t always update automatically. Check yours now to stay safe and browse without interruption!'), 'spread-plugincheck', 'sky mini') }}
```

Comment:
This is adding a new string - should we use a l10n tag here and only show the share cta for pages with the translation, or maybe there an existing fallback string we could use?